### PR TITLE
Feature/pdct 1094 move event types into the taxonomy

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

- Event types have been moved into the taxonomy (db_client version 3.6.1)
- Config tests updated

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: tests/integration_tests/config/test_config.py [hunk 1]
```diff
@@ -253,22 +250,6 @@ def test_config_documents(client: TestClient, data_db: Session, user_header_toke
     assert "Translation" in data["document"]["variants"]
 
 
-def test_config_events(client: TestClient, data_db: Session, user_header_token):
```
Comment (diversemix) [inline tests/integration_tests/config/test_config.py:256]: Why was this removed? Granted its not up to much, is it sufficiently covered elsewhere?

Comment (katybaulch) [reply tests/integration_tests/config/test_config.py:256]: Because the `events` key is no longer part of the config  - we've removed it because we've pushed it into the taxonomy

Comment (katybaulch) [reply tests/integration_tests/config/test_config.py:256]: The events key had a single key-value pair as a value: 
```
"events": {
  "event_types": {
    "Amended": "Amended Event", ...
  }
}
```

**Review by diversemix — APPROVED** [review]
lgtm
